### DISCUSSION TITLE
Changing the next version label

### DISCUSSION THREAD

**Discussion by Sotty75** [facebook/docusaurus#4931] [Q&A] [answered]
Hi,
i am enabling the versioning of documents in my docusaurus site, but in the example from the documentation it is not clear how to change the label for the next version:

```
{
            type: 'docsVersionDropdown',
              //// Optional
            position: 'right',
            // Add additional dropdown items at the beginning/end of the dropdown.
            dropdownItemsBefore: [],
            // Do not add the link active class when browsing docs.
            dropdownActiveClassDisabled: true,
            docsPluginId: 'default',
},

```
i am sure it is pretty simple, but i can't find examples.
Any suggestion?

**Comment by Siemienik** [comment, marked as answer]
It is able to set in docs plugin configuration, setting `versions`
```js
versions: {
          /*
          Example configuration: 
          current: {
            label: 'Android SDK v2.0.0 (WIP)',
            path: 'android-2.0.0',
          },
          '1.0.0': {
            label: 'Android SDK v1.0.0',
            path: 'android-1.0.0',
          },
          */
        },
```

https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs

**Reply by slorber** [reply]
Agree, we also have doc dedicated to this too:
https://docusaurus.io/docs/versioning#figure-out-the-behavior-for-the-current-version
